gossip/src/cluster_info.rs

```diff
@@ -2606,23 +2589,64 @@ impl ClusterInfo {
         Ok(())
     }
 
+    // Consumes packets received from the socket, deserializing, sanitizing and
+    // verifying them and then sending them down the channel for the actual
+    // handling of requests/messages.
+    fn run_consume(
```

Comment:
nit: rename to `run_socket_consume` to be more specific

Reply:
done